tests/component_handlers/kafka_connect/test_connect_wrapper.py

```diff
@@ -48,7 +47,7 @@ def test_should_through_exception_when_host_is_not_set(self):
             == "The Kafka Connect host is not set. Please set the host in the config."
         )
 
-    @patch("requests.post")
+    @patch("httpx.post")
```

Comment:
can this be replaced with HTTPXMock?

Reply:
httpx mock is for the responses but not for call assertions. For this we use the patch.